inst/qml/Prophet.qml

```diff
@@ -91,34 +91,35 @@ Form
 		{
 			value: "logistic"
 			label: qsTr("Logistic")
-		}
-	}
-
-	Group
-	{
-		DoubleField
-		{
-			name: "constantCapacity"
-			label: qsTr("Constant carrying capacity")
-			enabled: cap.count === 0
-			visible: growth.value === "logistic"
-			negativeValues: true
-			fieldWidth: 100
-		}
-		DoubleField
-		{
-			name: "constantMinimum"
-			label: qsTr("Constant saturating minimum")
-			enabled: floor.count === 0
-			visible: growth.value === "logistic"
-			negativeValues: true
-			fieldWidth: 100
+			childrenOnSameRow: true
+			
+			Group
+			{
+				DoubleField
+				{
+					name: "constantCapacity"
+					label: qsTr("Carrying capacity")
+					enabled: cap.count === 0
+					visible: growth.value === "logistic"
+					negativeValues: true
+					fieldWidth: 50
+				}
+				DoubleField
+				{
+					name: "constantMinimum"
+					label: qsTr("Saturating minimum")
+					enabled: floor.count === 0
+					visible: growth.value === "logistic"
+					negativeValues: true
+					fieldWidth: 50
+				}
+			}
```

Comment:
Yes, that does the trick! Now that it's nicely formatted, and the label is shortened, you actually have space to have the fields original width

```suggestion
			Group
			{
				DoubleField
				{
					name: "constantCapacity"
					label: qsTr("Carrying capacity")
					enabled: cap.count === 0
					visible: growth.value === "logistic"
					negativeValues: true
					fieldWidth: 100
				}
				DoubleField
				{
					name: "constantMinimum"
					label: qsTr("Saturating minimum")
					enabled: floor.count === 0
					visible: growth.value === "logistic"
					negativeValues: true
					fieldWidth: 100
				}
			}
```